Patches/Star Crafter's Armory/CMC300.xml

```diff
@@ -0,0 +1,193 @@
+					<value>
+						<ArmorRating_Blunt>150</ArmorRating_Blunt>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="CMC300a"]/costList/Plasteel</xpath>
+					<value>
+						<Plasteel>200</Plasteel>
+					</value>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="CMC300a"]/costList</xpath>
+					<value>
+						<DevilstrandCloth>80</DevilstrandCloth>
+					</value>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="CMC300a"]/equippedStatOffsets</xpath>
+					<value>
+						<CarryWeight>175</CarryWeight>
+						<CarryBulk>30</CarryBulk>
+					</value>
+				</li>
+
+
+				<!-- ==================== CMC300 helmet =====================-->
+
+				<!-- Carbon fiber, so lightweight but low HP and flammable-->
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="CMC300H" or defName="CMC300Ha" or defName="CMC300Hb"]/statBases</xpath>
+					<value>
+						<Bulk>10</Bulk>
+						<WornBulk>1</WornBulk>
+						<NightVisionEfficiency_Apparel>0.75</NightVisionEfficiency_Apparel>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="CMC300H" or defName="CMC300Ha" or defName="CMC300Hb"]/statBases/Mass</xpath>
+					<value>
+						<Mass>5</Mass>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="CMC300H" or defName="CMC300Ha" or defName="CMC300Hb"]/statBases/WorkToMake</xpath>
+					<value>
+						<WorkToMake>20000</WorkToMake>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="CMC300H" or defName="CMC300Ha" or defName="CMC300Hb"]/statBases/MaxHitPoints</xpath>
+					<value>
+						<MaxHitPoints>220</MaxHitPoints>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="CMC300H" or defName="CMC300Ha" or defName="CMC300Hb"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>24</ArmorRating_Sharp>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="CMC300H" or defName="CMC300Ha" or defName="CMC300Hb"]/statBases/ArmorRating_Blunt</xpath>
+					<value>
+						<ArmorRating_Blunt>60</ArmorRating_Blunt>
+					</value>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="CMC300H" or defName="CMC300Ha" or defName="CMC300Hb"]/equippedStatOffsets</xpath>
+					<value>
+						<PsychicSensitivity>-0.2</PsychicSensitivity>
+						<AimingAccuracy>0.15</AimingAccuracy>
+						<SmokeSensitivity>-1</SmokeSensitivity>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="CMC300H" or defName="CMC300Ha" or defName="CMC300Hb"]/equippedStatOffsets/ToxicSensitivity</xpath>
+					<value>
+						<ToxicSensitivity>-0.5</ToxicSensitivity>
```

Comment:
This tag is no longer used--it should be `<ToxicEnvironmentResistance>0.5</ToxicEnvironmentResistance>` instead.